gatsby-browser.js

```diff
@@ -46,8 +51,9 @@ const shouldUpdateScroll = ({ prevRouterProps, routerProps: { location } }) => {
 
   return (
```

Comment:
Don't need brackets

Reply:
This is a weird one, I noticed your linting isn't showing the same things as mine while pairing; this will be highlighted by prettier for me if I change it, are you using something else?

Reply:
I probably need to install the prettier extension